AnnoyancesFilter/sections/subscriptions.txt

```diff
@@ -40,6 +40,7 @@ mumsnet.com###main-container > div[\:class^="$store.adtech.reskinRenderedDesktop
 kinopoisk.ru##div[data-ui-id="post"] > div[style^="background-color:rgba(247, 247, 247, 1);"][style*="border-radius:10px"]
 guru99.com##.af-form-wrapper
 insights.sei.cmu.edu##.subscribe_small
+analyticsindiamag.com#?#div[data-elementor-type="single-post"] > section[data-element_type="section"]:not([data-settings]):has(> div.elementor-background-overlay + div.elementor-container)
```

Comment:
It looks dangerous.
Bot as I see [here](https://web.archive.org/web/20220129031109/https://analyticsindiamag.com/top-5-online-resources-to-learn-about-econometrics/) following rule can be used(at least it has own ID)
`analyticsindiamag.com##section[data-id="23d45d60"]`